pkg/minikube/update/update.go

```diff
@@ -90,11 +95,13 @@ automatically update from %s%s to %s%s now? [y/N] `,
 
 		if strings.ToLower(confirm) == "y" {
 			fmt.Printf("Updating to version %s\n", latestVersion)
-			updateBinary(latestVersion, downloadBinary, updateLinkPrefix, downloadLinkFormat)
+			updateBinary(latestVersion, downloadTarball, updateLinkPrefix, downloadLinkFormat)
```

Comment:
-1 to variable name. We are dealing with archives in general. We have tar as well as zip format.